adb-utils.spec

```diff
@@ -27,6 +27,8 @@ specific service and directly including it to kickstart file.
 
 %{__install} -pm644 services/openshift/openshift.service \
 %{buildroot}%{_unitdir}/openshift.service
+%{__install} -d -m 755 %{buildroot}%{_sysconfdir}/bash_completion.d/
+%{__install} -pm644 bash_completion/* %{buildroot}%{_sysconfdir}/bash_completion.d/
```

Comment:
s/-pm644/-p -m 644/ - to be consistent with the line above (30)? But it is also on line 28 which has already been there before.